specification/storage/resource-manager/Microsoft.Storage/stable/2019-06-01/storage.json

```diff
@@ -2637,14 +2637,47 @@
           "items": {
             "type": "string"
           },
-          "description": "An array of predefined enum values. Only blockBlob is supported."
+          "description": "An array of predefined enum values."
+        },
+        "blobIndexMatch": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/TagFilter"
+          },
+          "description": "An array of tag based filters, there can be at most 10 tag filters"
```

Comment:
description": "An array of blob index tag based filters, there can be at most 10 tag filters"